waspc/src/Wasp/Project/Analyze.hs

```diff
@@ -60,7 +75,7 @@ analyzeWaspProject waspDir options = do
         (Left prismaSchemaErrors, prismaSchemaWarnings) -> return (Left prismaSchemaErrors, prismaSchemaWarnings)
         -- NOTE: we are ignoring prismaSchemaWarnings if the schema was parsed successfully
         (Right prismaSchemaAst, _) ->
-          analyzeWaspFile prismaSchemaAst waspFilePath >>= \case
+          analyzeWaspFile waspDir prismaSchemaAst waspFilePath >>= \case
```

Comment:
`waspFilePath` -> `waspFile`?

Reply:
Hmm, I prefer keeping the `Path`. I think it's our general convention too.